web-registry/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -43,11 +44,16 @@ const sxs = {
 
 const Dashboard = (): JSX.Element => {
   const theme = useTheme();
-  const isIssuer = useQueryIfIssuer();
   const isCreditClassCreator = useQueryIfCreditClassCreator();
   const isCreditClassAdmin = useQueryIfCreditClassAdmin();
   const isProjectAdmin = useQueryIfProjectAdmin();
-  const walletContext = useWallet();
+  const { wallet } = useWallet();
+  const { data: projectsResponse } =
+    useEcocreditQuery<QueryProjectsByAdminResponse>({
+      query: 'projectsByAdmin',
+      params: { admin: wallet?.address },
+    });
+  const isIssuer = (projectsResponse?.projects.length ?? 0) > 0;
```

Comment:
could we just put this logic into the `useQueryIfIssuer` hook? i added this hook, so it's definitely done with a bit of naivety.

Reply:
Sure if using `projectsByAdmin` is sufficient as a replacement. I don't think `useQueryIfIssuer` implementation is naive, it's just that there's no other way to do it with the current ledger version unfortunatly.